packages/evolution-frontend/src/components/pageParts/validations/InterviewList.tsx

```diff
@@ -122,9 +124,35 @@ const InterviewList = <CustomSurvey, CustomHousehold, CustomHome, CustomPerson>(
                     {props.columns.map((column: any, j) =>
                         (column as any).enableSortBy === true ? (
                             <li key={`columnHeaderSort_${j}`}>
-                                <div key={`columnHeaderSort_${j}`} onClick={() => handleSortingChange(column.accessor)}>
+                                <span className="" key="sortColumnLabel">
                                     {column.label}
-                                </div>
+                                </span>
+                                <a
+                                    className={`faIconRight${
+                                        sortBy.length === 1 &&
+                                        sortBy[0].id === column.accessor.toString() &&
+                                        sortBy[0].desc !== true
+                                            ? ' _active-background _blue'
+                                            : ''
+                                    }`}
+                                    onClick={() => handleSortingChange(column.accessor)}
```

Comment:
`handleSortingChange` changes the current sorting order of the accessor, but here you have an arrow that should be 'asc' and another that should be 'desc'. If you click twice on the down arrow, there should be no change. You may have to send a second parameter in handleSortingChange with the direction or sorting and don't update if the order is the same

Reply:
fixed